src/lib.rs

```diff
@@ -16,6 +16,7 @@ use modules::{Module, ALL_MODULES};
 lazy_static::lazy_static! {
     pub static ref CTRLC_PRESSED: AtomicBool = AtomicBool::new(false);
     pub static ref SPEED_FACTOR: Mutex<f32> = Mutex::new(1.0);
+    pub static ref INSTANT_PRINT_LINES: Mutex<u32> = Mutex::new(0);
```

Comment:
Can't you use `AtomicU32` here instead?

Reply:
When I switch to `AtomicU32`, the value of `INSTANT_PRINT_LINES.load(Ordering::CstSeq)` in io.rs is always 0 regardless of whether I pass in the flag. Any suggestions of where to start looking to address this?

Reply:
Nevermind, got it. Will push a fix up shortly.